troubleshooting-additional-digit-appended-to-apk.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Additional Digit Appended to Version Code after Android Build
+description: The Android Version Code has the number two (2) appended to it after an application package cloud build.
+type: troubleshooting
+page_title: Wrong Version Code after Cloud Build
+slug: additional-digit-appended-to-apk
+position:
+tags: appbuilder, android, cordova
+teampulseid:
+ticketid: 1003785, 1015088, 1020035
+pitsid:
+
+---
+
+## Environment
+<table>
+  <tr>
+    <td>Service</td>
+    <td>Code, Distribution</td>	
+  </tr>
+  <tr>
+    <td>Feature</td>
+    <td>Android Cloud Build - Application package </td>	
+  </tr>
+  <tr>
+    <td>Mobile development type</td>
+    <td>Hybrid 
+  <tr>
+    <td>Mobile OS</td>
+    <td>Android</td>	
+  </tr>
+  <tr>
+    <td>Cordova framework version</td>
+    <td>Cordova 3.7.0 and above</td>	
+  </tr>
+</table>
+
+## Description
+There is an additional **2** appended in the `android:versionCode` property after you build a hybrid project for the Android platform. For example, if you have set the **Properties > Android > Version Code** to **1**, after the Cloud Build or Publish operation, the actual versionCode in the apk AndroidManifest.xml file will be **12**.
+
+## Steps to Reproduce
+1. Create a Cordova application.
+1. Verify that the **Version Code** property, under **Properties > Android** is set to **1**.
+1. Build the project for the Android platform as an application package.
+1. Check the built apk **AndroidManifest.xml** file. The `android:versionCode` is **12**.
```

Comment:
I have probably forgotten to mention this but we're using code formatting for file and folder names.

```
`AndroidManifest.xml`
```